internal/helm/release.go

```diff
@@ -29,45 +29,64 @@ import (
 	hmc "github.com/Mirantis/hmc/api/v1alpha1"
 )
 
-func ReconcileHelmRelease(
-	ctx context.Context,
+const (
+	DefaultReconcileInterval = 10 * time.Minute
+)
+
+type ReconcileHelmReleaseOpts struct {
+	Values            *apiextensionsv1.JSON
+	OwnerReference    *metav1.OwnerReference
+	ChartRef          *hcv2.CrossNamespaceSourceReference
+	ReconcileInterval *time.Duration
+	DependsOn         []meta.NamespacedObjectReference
+	TargetNamespace   string
+	CreateNamespace   bool
+}
+
+func ReconcileHelmRelease2(ctx context.Context,
```

Comment:
```suggestion
func ReconcileHelmRelease(ctx context.Context,
```

Reply:
LOL this somehow made its way into the PR